CHANGELOG.md

```diff
@@ -42,8 +42,8 @@ Please also have a look at our
 
 ### Fixed
 
+- Revert broken support for multiple comments (#740)
```

Comment:
The incorrect fix hasn't been included in a release yet (it wasn't in 8.6.0), so I don't think we need a changelog entry to say it's been taken out - just remove the one that was added for it.

Reply:
Done.